### PR TITLE
[JBPM-8476] Fix selectAll and scheduledStartTime format

### PR DESCRIPTION
**Pull request by ruromero**
Fix https://issues.jboss.org/browse/JBPM-8476

Signed-off-by: Ruben Romero Montes <rromerom@redhat.com>

### CODE REVIEW

**Comment by ruromero** [comment]
@jstastny-cz can you review, please?

File: process-migration-service/frontend/src/component/tabMigrationPlan/wizardExecuteMigration/PageMigrationRunningInstances.js [hunk 1]
```diff
@@ -266,7 +266,7 @@ export default class PageMigrationRunningInstances extends React.Component {
   };
   onRow = row => {
     const { selectedRows } = this.state;
-    const selected = selectedRows.indexOf(row.id) > -1;
+    const selected = selectedRows.indexOf(row.processInstanceId) > -1;
```
Comment (jstastny-cz) [inline process-migration-service/frontend/src/component/tabMigrationPlan/wizardExecuteMigration/PageMigrationRunningInstances.js]: const selected = selectedRows.indexOf(row.processInstanceId) > -1;
```suggestion
    const selected = selectedRows.includes(row.processInstanceId);
```

File: process-migration-service/frontend/src/component/tabMigrationPlan/wizardExecuteMigration/PageMigrationRunningInstances.js [hunk 2]
```diff
@@ -279,10 +279,15 @@ export default class PageMigrationRunningInstances extends React.Component {
...
       const updatedSelections = [
-        ...new Set([...currentRows.map(r => r.id), ...selectedRows])
+        ...new Set([
+          ...currentRows.map(r => r.processInstanceId),
+          ...selectedRows
+        ])
       ];
       const updatedRows = rows.map(r =>
-        updatedSelections.indexOf(r.id) > -1 ? this.selectRow(r) : r
+        updatedSelections.indexOf(r.processInstanceId) > -1
```
Comment (jstastny-cz) [inline process-migration-service/frontend/src/component/tabMigrationPlan/wizardExecuteMigration/PageMigrationRunningInstances.js]: again indexOf -> includes ?

+ multiple occurrences in the PR..

**Comment by jstastny-cz** [comment]
@ruromero is the following expected when running tests via npm test ?

```
Deprecation warning: value provided is not in a recognized RFC2822 or ISO format. moment construction falls back to js Date(), which is not reliable across all browsers and versions. Non RFC2822/ISO date formats are discouraged and will be removed in an upcoming major release. Please refer to http://momentjs.com/guides/#/warnings/js-date/ for more info.
      Arguments: 
      [0] _isAMomentObject: true, _isUTC: false, _useUTC: false, _l: undefined, _i: 2019-7-17T20:59:22.128Z, _f: undefined, _strict: undefined, _locale: [object Object]
      Error: 
          at Function.createFromInputFallback (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/moment/moment.js:320:98)
          at configFromString (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/moment/moment.js:2385:15)
          at configFromInput (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/moment/moment.js:2611:13)
          at prepareConfig (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/moment/moment.js:2594:13)
          at createFromConfig (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/moment/moment.js:2561:44)
          at createLocalOrUTC (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/moment/moment.js:2648:16)
          at createLocal (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/moment/moment.js:2652:16)
          at hooks (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/moment/moment.js:12:29)
          at PageMigrationScheduler.scheduledStartTime [as formatScheduledStartTime] (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/src/component/tabMigrationPlan/wizardExecuteMigration/PageMigrationScheduler.js:87:12)
          at new formatScheduledStartTime (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/src/component/tabMigrationPlan/wizardExecuteMigration/PageMigrationScheduler.js:21:14)
          at constructClassInstance (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/react-dom/cjs/react-dom.development.js:11361:18)
          at updateClassComponent (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/react-dom/cjs/react-dom.development.js:14687:5)
          at beginWork (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/react-dom/cjs/react-dom.development.js:15644:16)
          at performUnitOfWork (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/react-dom/cjs/react-dom.development.js:19312:12)
          at workLoop (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/react-dom/cjs/react-dom.development.js:19352:24)
          at renderRoot (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/react-dom/cjs/react-dom.development.js:19435:7)
          at performWorkOnRoot (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/react-dom/cjs/react-dom.development.js:20342:7)
          at requestWork (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/react-dom/cjs/react-dom.development.js:20090:7)
          at scheduleWork (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/react-dom/cjs/react-dom.development.js:19911:5)
          at scheduleRootUpdate (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/react-dom/cjs/react-dom.development.js:20572:3)
          at updateContainerAtExpirationTime (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/react-dom/cjs/react-dom.development.js:20600:10)
          at updateContainer (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/react-dom/cjs/react-dom.development.js:20657:10)
          at ReactRoot.Object.<anonymous>.ReactRoot.render (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/react-dom/cjs/react-dom.development.js:20953:3)
          at /home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/react-dom/cjs/react-dom.development.js:21090:14
          at unbatchedUpdates (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/react-dom/cjs/react-dom.development.js:20454:14)
          at legacyRenderSubtreeIntoContainer (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/react-dom/cjs/react-dom.development.js:21086:5)
          at Object.render (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/react-dom/cjs/react-dom.development.js:21155:12)
          at /home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/react-testing-library/dist/index.js:83:25
          at Object.batchedUpdates$1 [as unstable_batchedUpdates] (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/react-dom/cjs/react-dom.development.js:20439:12)
          at act (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/react-dom/cjs/react-dom-test-utils.development.js:1161:27)
          at render (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/react-testing-library/dist/index.js:79:26)
          at Object.<anonymous>.test (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/src/__tests__/tabMigrationPlan/PageMigrationScheduler.test.js:66:30)
          at Object.asyncJestTest (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/jest-jasmine2/build/jasmineAsyncInstall.js:102:37)
          at resolve (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/jest-jasmine2/build/queueRunner.js:43:12)
          at new Promise (<anonymous>)
          at mapper (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/jest-jasmine2/build/queueRunner.js:26:19)
          at promise.then (/home/jstastny/gitrepo/droolsjbpm-integration/process-migration-service/frontend/node_modules/jest-jasmine2/build/queueRunner.js:73:41)
          at <anonymous>
          at process._tickCallback (internal/process/next_tick.js:189:7)

```

**Comment by ruromero** [comment]
> @ruromero is the following expected when running tests via npm test ?

No, thanks for noticing. Fixed.

**Comment by ruromero** [comment]
jenkins retest this

**Comment by ruromero** [comment]
jenkins retest this

**Comment by jstastny-cz** [comment]
jenkins please retest this

**Comment by jstastny-cz** [comment]
jenkins please retest this

**Comment by jstastny-cz** [comment]
jenkins please retest this

**Pull request merged**